index.html

```diff
@@ -55,17 +55,18 @@ <h1>Quiz - App</h1>
     </main>
 
     <footer class="footer">
-      <nav>
-        <a class="active">
-          <img src="./CSS/icons/house-white-50.png" alt="link zur Startseite" />
+      <nav class="nav-icons">
+        <a href="./index.html">
```

Comment:
This is actually a single 'navigation-item' I would suggest to find a good class name!
Please consider also to add the BEM modifier to it when 'active' (ie. `navigation-item--active`) instead of settin git on the `<img>`inside of it.